website/docs/support/github-101.md

```diff
@@ -0,0 +1,91 @@
+---
+id: github-101
+title: GitHub 101
+---
+
+Welcome to GitHub 101!
+
+Here will be the guide to all the steps you need to follow in order to be able to participate in the challenge.
+
+We use GitHub. There are all of our repositories and where you can access the files needed to being working on the project and Amiga.
+
+Prerequisites
+1) First you will need to have an Github account
+- To create on you can go [here](https://www.github.com)
+
+Once you do that you will need to generate a personal token:
+
+## Personal Access Token
+1. Verify your email address, if it hasn't been verified yet.
+
+2. In the upper-right corner of any page, click your profile photo, then click Settings.
+![userbar-account-settings](https://user-images.githubusercontent.com/64480560/216469067-dd3a5214-a4c8-43c2-919b-2905bcb967ed.png)
+
+3. In the left sidebar, click  Developer settings.
+
+4. In the left sidebar, under  Personal access tokens, click Tokens (classic).
+
+5. Select Generate new token, then click Generate new token (classic).
+
+6. Give your token a descriptive name.
+
+:::tip
+To give your token an expiration, select the Expiration drop-down menu, then click a default or use the calendar picker.
+:::
+
+
+8. Select the scopes you'd like to grant this token.
+
+9.  To use your token to access repositories from the command line, select repo. A token with no assigned scopes can only access public information. For more information, see "Available scopes".
+
+10. Once that has been created you can then begin cloning the repositories from farm-ng!
+
+
+## Editing and Contributing
+
+Here you will learn about a good workflow to have when contributing to code.
+
+Before anything be sure to clone the respective repository. You can do this following the listed commands below.
+
+```bash
+cd ~ #home directory
+git clone <RESPECTIVE REPOSITORY>
+cd <RESPECTIVE REPOSITORY>
+```
+Once you have cloned into that repository, you will need to create your own branch. This can be done with these commands:
+
+```bash
+cd <RESPECTIVE REPOSITORY>
+git checkout -b "insert name of branch here"
+```
+
+First you have completed your code and you are ready to make a push, you should follow this workflow.
+
+
+Pre-commit your code
+```
+git add -p # Add the changes I want
+pre-commit run --all-files
```

Comment:
this is not really necessary if we do install pre-commit before. I'd add a previous step `pre-commit install`